sdcm/nemesis.py

```diff
@@ -525,6 +525,11 @@ def wait_for_old_node_to_removed():
         finally:
             new_node.running_nemesis = None
 
+    def disrupt_kill_scylla(self):
+        self._set_current_disruption('ScyllaKillMonkey %s' % self.target_node)
+        self.target_node.wait_db_up(timeout=400)
```

Comment:
Why do you need to "wait_db_up" before you killed it?

Reply:
move it to after the _kill_scylla_daemon

Reply:
kill_scylla_deamon already has it, will remove it